### ISSUE TITLE
[1.20.1] Canary crash with Connector + C2ME

### ISSUE THREAD

**Issue by Reshy** [AbdElAziz333/Canary#231]
### Version

> canary-mc1.20.1-0.3.3.jar
> fabric-api-0.92.2+1.11.8+1.20.1.jar
> Connector-1.0.0-beta.44+1.20.1.jar
> c2me-fabric-mc1.20.1-0.2.0+alpha.11.8.jar

### Reproduction Steps

> Example:
> 1. Load the game with the given mods
> 2. Crash:  @Redirect conflict. Skipping canary.mixins.json:alloc.nbt.CompoundTagMixin from mod canary->@Redirect::removeOldMapAlloc()Ljava/util/HashMap; with priority 1000, already redirected by c2me-opts-allocs.mixins.json:MixinNbtCompound from mod c2me_opts_allocs->@Redirect::redirectNewHashMap()Ljava/util/HashMap; with priority 1100

### Crash Report File

> None, logs:  https://pastebin.com/yST71nZG

### Additional Information

> Crash does not occur on Radium

**Comment by NBZion** [comment]
are you running on forge or fabric? because canary is a forge only mod

**Comment by Reshy** [comment]
Yes, this is canary being used on Forge in a connector environment.

**Comment by NBZion** [comment]
try removing c2me because maybe it's causing issues to radium and not sinytra?????

**Comment by MrCubicYT** [comment]
I found the fix! It crashes because they are both trying to optimise the same thing I presume.

Add this line to canary's config file:
`mixin.alloc.nbt=false`

**Comment by MrCubicYT** [comment]
Another fix, to actually generate the world!

Add this to the config file as well:
`mixin.chunk.replace_streams=false`

**Comment by the80hd** [comment]
@MrCubicYT bro thank you so much! This worked for me. Been scratching my head on this for like a month.

**Comment by Slimmeyy** [comment]
Been a few months but can confirm this is still the case. Radium and Palladium (both also Forge ports of Lithium) don't have this issue and launch and create/load worlds just fine. 

Can also confirm that Bliss' mixin suggestions still work. Not sure if it'll help anyone, but to make it extra clear:
- The config can be found at /config/canary.properties
- Add the lines `mixin.alloc.nbt=false` and `mixin.chunk.replace_streams=false` below the notice
